traffic_ops/testing/api/v3/topologies_test.go

```diff
@@ -61,6 +62,19 @@ func CreateTestTopologies(t *testing.T) {
 	}
 }
 
+func GetTestTopologies(t *testing.T) {
```

Comment:
For posterity, we should also add this test to the v4 tests.

Reply:
Done